modules/apps/analytics/analytics-settings-web/package.json

```diff
@@ -0,0 +1,11 @@
+{
+	"main": "js/index.es.js",
```

Comment:
Follow this https://github.com/pt-liferay-solutions/liferay-portal/pull/2143#pullrequestreview-1106273108, it could be good check the `*.es.js `extension here too.